github-dark.css

```diff
@@ -500,6 +500,10 @@
   .discussion-topic-header, .previewable-comment-form .tabnav {
     border-radius: 3px 3px 0 0 !important;
   }
+  /* star count and color in repo lists */
+  .mini-repo-list-item .stars {
+	  color: #777;
+  }
```

Comment:
Can you add your selector to [this line](https://github.com/StylishThemes/GitHub-Dark/blob/88e24df93005d56afde31d03aef6f172dfc799ed/github-dark.css#L1933) instead? We try to conserve space by grouping similar colors together and I think `#7b7b7b` is reasonably close to your `#7777`.